backend/spherre/app/service/member.py

```
@@ -0,0 +1,28 @@
+    def update_member_email(member_address: str, new_email: str) -> Member | None:
+        """
+        Update the email of a member
+        """
+        member = Member.query.filter_by(address=member_address).one_or_none()
+        if not member:
+            return None
+        member.email = new_email
+        member.save()
+        return member
+
+    def get_or_create_member(member_address: str, **kwargs) -> Member:
```

Pylint (3.3.7)

[error] Method 'get_or_create_member' should have "self" as first argument

(E0213)